modules/viewing-distributed-workload-status.adoc

```diff
@@ -0,0 +1,56 @@
+:_module-type: PROCEDURE
+
+[id="viewing-project-metrics-for-distributed-workloads_{context}"]
+= Viewing distributed workload status
```

Comment:
Just wondering if a better module title could be used here:

"Monitoring distributed workloads"?

Reply:
I think that's a good title for a parent section for both of the new metrics modules, I've added it accordingly.